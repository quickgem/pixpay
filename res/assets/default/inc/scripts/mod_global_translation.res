TK_CONST_DATA_ALIGN(const unsigned char script_mod_global_translation[]) = {
0x07,0x00,0x01,0x01,0x37,0x01,0x00,0x00,0x00,0x00,0x00,0x00,0x6d,0x6f,0x64,0x5f,0x67,0x6c,0x6f,0x62,
0x61,0x6c,0x5f,0x74,0x72,0x61,0x6e,0x73,0x6c,0x61,0x74,0x69,0x6f,0x6e,0x00,0x00,0x00,0x00,0x00,0x00,
0x00,0x00,0x00,0x00,0x66,0x75,0x6e,0x63,0x74,0x69,0x6f,0x6e,0x20,0x47,0x45,0x54,0x5f,0x4d,0x55,0x4c,
0x54,0x49,0x5f,0x54,0x45,0x58,0x54,0x28,0x73,0x74,0x72,0x29,0x20,0x7b,0x0d,0x0a,0x20,0x20,0x20,0x20,
0x69,0x66,0x20,0x28,0x21,0x73,0x74,0x72,0x29,0x20,0x72,0x65,0x74,0x75,0x72,0x6e,0x20,0x22,0x22,0x3b,
0x0d,0x0a,0x20,0x20,0x20,0x20,0x6c,0x65,0x74,0x20,0x72,0x65,0x74,0x20,0x3d,0x20,0x54,0x6f,0x73,0x2e,
0x53,0x79,0x73,0x47,0x65,0x74,0x53,0x74,0x72,0x69,0x6e,0x67,0x28,0x73,0x74,0x72,0x29,0x3b,0x0d,0x0a,
0x20,0x20,0x20,0x20,0x63,0x6f,0x6e,0x73,0x6f,0x6c,0x65,0x2e,0x6c,0x6f,0x67,0x28,0x22,0x47,0x45,0x54,
0x5f,0x4d,0x75,0x6c,0x74,0x69,0x5f,0x54,0x45,0x58,0x54,0x20,0x30,0x30,0x30,0x30,0x3d,0x3d,0x3d,0x3d,
0x3d,0x3d,0x3d,0x3d,0x3e,0x20,0x22,0x2c,0x20,0x72,0x65,0x74,0x20,0x3f,0x20,0x72,0x65,0x74,0x2e,0x63,
0x6f,0x64,0x65,0x20,0x3a,0x20,0x22,0x22,0x2c,0x20,0x72,0x65,0x74,0x20,0x3f,0x20,0x72,0x65,0x74,0x2e,
0x64,0x61,0x74,0x61,0x20,0x3a,0x20,0x22,0x22,0x29,0x3b,0x0d,0x0a,0x20,0x20,0x20,0x20,0x69,0x66,0x20,
0x28,0x21,0x72,0x65,0x74,0x20,0x7c,0x7c,0x20,0x72,0x65,0x74,0x2e,0x63,0x6f,0x64,0x65,0x20,0x21,0x3d,
0x3d,0x20,0x30,0x20,0x7c,0x7c,0x20,0x21,0x72,0x65,0x74,0x2e,0x64,0x61,0x74,0x61,0x29,0x20,0x72,0x65,
0x74,0x75,0x72,0x6e,0x20,0x22,0x22,0x3b,0x0d,0x0a,0x20,0x20,0x20,0x20,0x72,0x65,0x74,0x75,0x72,0x6e,
0x20,0x72,0x65,0x74,0x2e,0x64,0x61,0x74,0x61,0x3b,0x0d,0x0a,0x7d,0x0d,0x0a,0x65,0x78,0x70,0x6f,0x72,
0x74,0x73,0x2e,0x47,0x45,0x54,0x5f,0x4d,0x55,0x4c,0x54,0x49,0x5f,0x54,0x45,0x58,0x54,0x20,0x3d,0x20,
0x47,0x45,0x54,0x5f,0x4d,0x55,0x4c,0x54,0x49,0x5f,0x54,0x45,0x58,0x54,0x3b,0x00,0x00,0x00,0x00,};/*359*/
