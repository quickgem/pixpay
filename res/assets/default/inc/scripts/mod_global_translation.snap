TK_CONST_DATA_ALIGN(const unsigned char script_mod_global_translation[]) = {
0x07,0x00,0x01,0x01,0x94,0x01,0x00,0x00,0x00,0x00,0x00,0x00,0x6d,0x6f,0x64,0x5f,0x67,0x6c,0x6f,0x62,
0x61,0x6c,0x5f,0x74,0x72,0x61,0x6e,0x73,0x6c,0x61,0x74,0x69,0x6f,0x6e,0x00,0x00,0x00,0x00,0x00,0x00,
0x00,0x00,0x00,0x00,0x4a,0x52,0x52,0x59,0x46,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x08,0x01,0x00,0x00,
0x01,0x00,0x00,0x00,0x18,0x00,0x00,0x00,0x07,0x00,0x01,0x00,0x10,0x30,0x03,0x00,0xc8,0x27,0x47,0x00,
0x01,0x01,0x04,0x05,0x07,0x00,0x00,0x00,0x47,0x01,0x00,0x00,0x47,0x04,0x00,0x00,0x38,0x00,0x00,0x00,
0x3d,0x01,0x14,0xe7,0x02,0x00,0x10,0x04,0x03,0x3d,0x01,0x3b,0x02,0x33,0xd8,0x56,0x60,0x28,0x47,0x00,
0x47,0x09,0x00,0x00,0x06,0x00,0x01,0x00,0x30,0x10,0x03,0x02,0xc8,0x27,0x47,0x00,0x03,0x03,0x04,0x05,
0x47,0x05,0x00,0x00,0x30,0x00,0x00,0x00,0x53,0x04,0x02,0x2f,0x00,0x03,0x02,0xdc,0x56,0x00,0x00,0x00,
0x5b,0x82,0x2e,0x02,0x58,0x28,0x47,0x00,0x47,0x04,0x00,0x00,0x11,0x00,0x01,0x00,0x30,0x10,0x06,0x01,
0xc8,0x27,0x47,0x00,0x02,0x04,0x0b,0x0b,0x47,0x07,0x00,0x00,0x07,0x08,0x00,0x00,0x47,0x09,0x00,0x00,
0x87,0x09,0x00,0x00,0x47,0x0b,0x00,0x00,0x07,0x0c,0x00,0x00,0x07,0x10,0x00,0x00,0xc7,0x10,0x00,0x00,
0x00,0x00,0x00,0x00,0x61,0x00,0x11,0x04,0x57,0x04,0x41,0x02,0x05,0x2c,0x00,0xd4,0xeb,0x01,0x41,0x03,
0x06,0x2d,0x07,0x01,0x11,0x07,0x3c,0x01,0x08,0x01,0x04,0x2c,0x04,0x2c,0x01,0x11,0x07,0x3c,0x01,0x09,
0x01,0x04,0x2c,0x04,0xc7,0x03,0x61,0x01,0x1d,0x0d,0x3c,0x01,0x08,0x7b,0x0a,0x1d,0x06,0x3c,0x01,0x09,
0x60,0x11,0x04,0x57,0x04,0x3c,0x01,0x09,0x55,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x0d,0x82,0x51,0x02,
0x80,0x28,0x47,0x00,0x47,0x05,0x00,0x00,0x07,0x00,0x6d,0x6f,0x64,0x75,0x6c,0x65,0x73,0x00,0x16,0x00,
0x6d,0x6f,0x64,0x5f,0x67,0x6c,0x6f,0x62,0x61,0x6c,0x5f,0x74,0x72,0x61,0x6e,0x73,0x6c,0x61,0x74,0x69,
0x6f,0x6e,0x06,0x00,0x6d,0x6f,0x64,0x75,0x6c,0x65,0x0e,0x00,0x47,0x45,0x54,0x5f,0x4d,0x55,0x4c,0x54,
0x49,0x5f,0x54,0x45,0x58,0x54,0x03,0x00,0x54,0x6f,0x73,0x00,0x07,0x00,0x63,0x6f,0x6e,0x73,0x6f,0x6c,
0x65,0x00,0x00,0x00,0x0c,0x00,0x53,0x79,0x73,0x47,0x65,0x74,0x53,0x74,0x72,0x69,0x6e,0x67,0x03,0x00,
0x6c,0x6f,0x67,0x00,0x1d,0x00,0x47,0x45,0x54,0x5f,0x4d,0x75,0x6c,0x74,0x69,0x5f,0x54,0x45,0x58,0x54,
0x20,0x30,0x30,0x30,0x30,0x3d,0x3d,0x3d,0x3d,0x3d,0x3d,0x3d,0x3d,0x3e,0x20,0x00,0x04,0x00,0x63,0x6f,
0x64,0x65,0x04,0x00,0x64,0x61,0x74,0x61,0x00,0x00,0x00,0x00,};/*452*/
